indico/modules/events/export.yaml

```diff
@@ -221,6 +221,12 @@ export:
     fks_out:
       - event_role_id
 
+  MenuEntryPrincipal:
+    skipif: ROW.type.name not in ('user', 'event_role', 'registration_form')
+    fks_out:
+      - registration_form_id
+      - event_role_id
```

Comment:
This ensures ordering in the export file: registration forms and event roles must be imported first, because it's impossible to create the menu entry principals without those if they are referenced (they can't be set lazily later, because an event role principal cannot have null event role id)